components/MarkdownEditor.tsx

```diff
@@ -152,7 +149,9 @@ export default function MarkdownEditor(props: {
               },
             }}
             style={{
-              border: editMode ? "2px solid #007079" : "1px solid #ECECEC",
+              border: editMode
+                ? "2px solid " + colors.EQUINOR_PROMINENT
+                : "1px solid " + colors.EQUINOR_DISABLED,
```

Comment:
Please use [Template literals (Template strings)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) when concatenating strings and variables 🙌

Reply:
Fixed